addons/packages/pinniped/0.22.0/bundle/.imgpkg/images.yml

```diff
@@ -0,0 +1,25 @@
+---
+apiVersion: imgpkg.carvel.dev/v1alpha1
+images:
+- annotations:
+    kbld.carvel.dev/id: docker.io/getpinniped/pinniped-server:v0.12.1
+    kbld.carvel.dev/origins: |
+      - resolved:
+          tag: v0.12.1
```

Comment:
We need to update this or we will still get `v0.12.1`.

Reply:
Updated and pushed.